rest/model/task_annotations.go

```diff
@@ -3,57 +3,50 @@ package model
 import (
 	"time"
 
-	"github.com/evergreen-ci/birch"
 	"github.com/evergreen-ci/evergreen/model/task_annotations"
 )
 
 type APITaskAnnotation struct {
-	Id              *string             `bson:"_id" json:"id"`
-	TaskId          *string             `bson:"task_id" json:"task_id"`
-	TaskExecution   int                 `bson:"task_execution" json:"task_execution"`
-	Note            *string             `bson:"note" json:"note"`
-	Issues          []APIIssueLink      `bson:"issues" json:"issues"`
-	SuspectedIssues []APIIssueLink      `bson:"suspected_issues" json:"suspected_issues"`
-	Source          APIAnnotationSource `bson:"source" json:"source"`
-	Metadata        *birch.Document     `bson:"metadata" json:"metadata"`
+	Id             *string       `bson:"_id" json:"id"`
+	TaskId         *string       `bson:"task_id" json:"task_id"`
+	TaskExecution  int           `bson:"task_execution" json:"task_execution"`
+	APIAnnotation  APIAnnotation `bson:"api_annotation,omitempty" json:"api_annotation,omitempty"`
+	UserAnnotation APIAnnotation `bson:"user_annotation,omitempty" json:"user_annotation,omitempty"`
```

Comment:
do we not care about metadata here?

Reply:
hmm, Actually, I should add it. We don't care about metadata for the UI, but we do for the restAPI